src/test/integration/checkResponseCode.py

```diff
@@ -0,0 +1,65 @@
+import json
+import pycurl
+import urllib
+import StringIO
+
```

Comment:
Let's add a readme so that a new dev knows what command to use to run the tests. Another thing to keep in mind is whether some tests can be run in parallel. Maybe don't add support, but mention them as todo. This is something we could do to speed up integration tests in the future.